x-pack/plugins/security/server/routes/api/public/roles/get.js

```diff
@@ -42,10 +42,17 @@ export function initGetRolesApi(server, callWithRequest, routePreCheckLicenseFn,
       value: roleKibanaApplications.map(({ resources, privileges }) => {
       // if we're dealing with a global entry, which we've ensured above is only possible if it's the only item in the array
         if (resources.length === 1 && resources[0] === GLOBAL_RESOURCE) {
+          const reservedPrivileges = privileges.filter(privilege => PrivilegeSerializer.isSerializedReservedPrivilege(privilege));
           const basePrivileges = privileges.filter(privilege => PrivilegeSerializer.isSerializedGlobalBasePrivilege(privilege));
-          const featurePrivileges = privileges.filter(privilege => !PrivilegeSerializer.isSerializedGlobalBasePrivilege(privilege));
+          const featurePrivileges = privileges.filter(privilege =>
+            !PrivilegeSerializer.isSerializedGlobalBasePrivilege(privilege) &&
+            !PrivilegeSerializer.isSerializedReservedPrivilege(privilege)
```

Comment:
If `isSerializedGlobalBasePrivilege` is correct (see my earlier comment), then we shouldn't also need to check `isSerializedReservedPrivilege`, because reserved privileges are being treated like global base privileges (which doesn't feel right IMO)